cmd/ocm/fail/job/job.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package job
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+)
+
+// Cmd Constant:
+var Cmd = &cobra.Command{
+	Use:     "failure QUEUE_NAME JOB_ID RECEIPT_ID FAILURE_REASON",
```

Comment:
Is this accurate?  The resource is `fail`, so it should be `fail job QUEUE_NAME...` not `fail failure QUEUE_NAME`

I think you have this correct for the success resource.

Reply:
Fixed

Reply:
done